src/main/java/org/vanilladb/core/query/planner/Verifier.java

```diff
@@ -66,20 +56,35 @@ public static void verifyQueryData(QueryData data, Transaction tx) {
 			}
 		}
 
+		boolean allFields = false;
+
 		// examine the projecting field name
 		for (String fldName : data.projectFields()) {
+			if (fldName.equals("*")) {

```

Comment:
I think there should also be a constant for the wildcard in this class like how we did in `Lexer`.